chapters/en/Unit 2 - Convolutional Neural Networks/mobilenet.mdx

```diff
@@ -0,0 +1,173 @@
+# MobileNet
+
+MobileNet is a type of neural network architecture designed for mobile devices. It was developed by Google's research team and first introduced in 2017. The primary goal of MobileNet is to provide high-performance, low-latency image classification and object detection on smartphones, tablets, and other resource-constrained devices.  
+  
+MobileNet achieves this by using depthwise separable convolutions, which are a more efficient alternative to standard convolutions. Depthwise separable convolutions break down the computation into two separate steps: depthwise convolution followed by pointwise convolution. This results in a significant reduction of parameters and computational complexity, allowing MobileNet to run efficiently on mobile devices.  
+
+## Convolution Types on MobileNet
+
+By replacing regular convolutional layers with these depthwise separable convolutions and pointwise convolutions, MobileNet achieves high accuracy while minimizing computational overhead, making it well-suited for mobile devices and other resource-limited platforms. There are two types of convolutions used in MobileNet:
```

Comment:
It is either only "depthwise separable convolutions" or "depthwise convolutions and pointwise convolutions".